agent/pkg/app/main.go

```diff
@@ -86,6 +86,11 @@ func ConfigureBasenineServer(host string, port string, dbSize int64, logLevel lo
 			}
 		}
 	}
+
+	// Set the insertion filter that comes from the config
+	if err := basenine.InsertionFilter(host, port, insertionFilter); err != nil {
```

Comment:
I tried to run and it is working very well.
just didn't saw a case that this log is happening.. 
even when I use invalid filter it seems like it is not crashing (but it is not showing any information so it is filtered)
am I missing anything here?